### DISCUSSION TITLE
Welcome to DeveloperCommunity Discussions!

### DISCUSSION THREAD

**Discussion by rectinajhua** [deeper-chain/DeveloperHub#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by edwardgrubb3rd** [comment]
I want to participate on this project but it seems the developer community has died.

Currently I bother Eric Ma when I find a bug or a high-priority exploit. All the code I've written I cannot publish because developers are missing access to basic contributing prerequisites and have zero guidance.

I'd go as far as saying I believe I have spent more hours studying the Deeper Chain but I seem to be the only one interested. 

If you looking for a skilled developer who already knows your entire codebase and are capable of running the developer community please let me know. If not, I'll find a new project to contribute.
